scipy/optimize/lbfgsb/lbfgsb.pyf

```diff
@@ -20,6 +20,7 @@ python module _lbfgsb ! in
             logical dimension(4),intent(inout) :: lsave
             integer dimension(44),intent(inout) :: isave
             double precision dimension(29),intent(inout) :: dsave
+            integer intent(in) :: maxls
```

Comment:
I'd add `check(maxls > 0)` here. Or even do basic sanity checking at the python level.